### PR TITLE
Fortinet Fortiweb XDM Mapping

### PR DESCRIPTION
**Pull request by eepstain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-3315)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FortinetFortiwebVM/README.md [hunk 1]
```diff
@@ -11,3 +22,94 @@ Fortiweb VM integration enables you to manage WAF policies and block cookies, UR
 - Retrieve information and status of the systems.
 
 This pack contains an integration, whose main purpose is to perform controlled changes on hosted web applications.
+
+---
+
+## Collect Events from Vendor
+In order to receive logs, use the [Broker VM](#broker-vm) option. <br>
+For Traffic logs type via Fortiweb Cloud, you are required to send the logs with [Amazon S3](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Ingest-Generic-Logs-from-Amazon-S3) services.
```
Comment (ShirleyDenkberg) [inline Packs/FortinetFortiwebVM/README.md:30]:
```suggestion
For Traffic logs via Fortiweb Cloud, you are required to send the logs with [Amazon S3](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Ingest-Generic-Logs-from-Amazon-S3) services.
```

File: Packs/FortinetFortiwebVM/README.md [hunk 2]
```diff
@@ -11,3 +22,94 @@ Fortiweb VM integration enables you to manage WAF policies and block cookies, UR
...
+**_Traffic_ Logs (AWS S3)**
+* [How-To export Attack and Traffic logs to a log server](https://docs.fortinet.com/document/fortiweb-cloud/23.3.0/user-guide/367276/audit-logs).
+1. Go to **Log Settings**.
+2. Enable **Traffic Log Export**.
+3. Configure the following mandatory settings: <br>
+
+| Field Name | Value |
+| :---------------- | :------: |
+| Server Type | AWS S3 |
+| Bucket Name | Enter the AWS S3 bucket name |
```
Comment (ShirleyDenkberg) [inline Packs/FortinetFortiwebVM/README.md:76]:
```suggestion
| Bucket Name | Enter the AWS S3 bucket name. |
```

File: Packs/FortinetFortiwebVM/README.md [hunk 3]
```diff
@@ -11,3 +22,94 @@ Fortiweb VM integration enables you to manage WAF policies and block cookies, UR
...
+---
+
+## Collect Events from Vendor
+In order to receive logs, use the [Broker VM](#broker-vm) option. <br>
+For Traffic logs type via Fortiweb Cloud, you are required to send the logs with [Amazon S3](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Ingest-Generic-Logs-from-Amazon-S3) services.
+
+### Fortiweb Cloud
+
+**_Audit_ Logs**
+* [How-To export Audit logs to a log server](https://docs.fortinet.com/document/fortiweb-cloud/23.3.0/user-guide/367276/audit-logs).
```
Comment (ShirleyDenkberg) [inline Packs/FortinetFortiwebVM/README.md:35]:
```suggestion
* [How to export Audit logs to a log server](https://docs.fortinet.com/document/fortiweb-cloud/23.3.0/user-guide/367276/audit-logs).
```

File: Packs/FortinetFortiwebVM/README.md [hunk 4]
```diff
@@ -11,3 +22,94 @@ Fortiweb VM integration enables you to manage WAF policies and block cookies, UR
...
+| IP/Domain and Port | <IP> and <Port>  |
+| Protocol | TCP |
+| Log Format | CEF |
+
+4. Click **Save**. 
+
+***
+
+**_Attack_ Logs**
+* [How-To export Attack and Traffic logs to a log server](https://docs.fortinet.com/document/fortiweb-cloud/23.3.0/user-guide/367276/audit-logs).
```
Comment (ShirleyDenkberg) [inline Packs/FortinetFortiwebVM/README.md:52]:
```suggestion
* [How to export Attack and Traffic logs to a log server](https://docs.fortinet.com/document/fortiweb-cloud/23.3.0/user-guide/367276/audit-logs).
```

File: Packs/FortinetFortiwebVM/README.md [hunk 5]
```diff
@@ -11,3 +22,94 @@ Fortiweb VM integration enables you to manage WAF policies and block cookies, UR
...
+| :---------------- | :------: |
+| Server Type | Syslog |
+| IP/Domain and Port | <IP> and <Port>  |
+| Protocol | TCP |
+| Log Format | CEF |
+
+5. Click **OK**. 
+
+**_Traffic_ Logs (AWS S3)**
+* [How-To export Attack and Traffic logs to a log server](https://docs.fortinet.com/document/fortiweb-cloud/23.3.0/user-guide/367276/audit-logs).
```
Comment (ShirleyDenkberg) [inline Packs/FortinetFortiwebVM/README.md:68]:
```suggestion
* [How to export Attack and Traffic logs to a log server](https://docs.fortinet.com/document/fortiweb-cloud/23.3.0/user-guide/367276/audit-logs).
```

File: Packs/FortinetFortiwebVM/README.md [hunk 6]
```diff
@@ -11,3 +22,94 @@ Fortiweb VM integration enables you to manage WAF policies and block cookies, UR
...
+| Secret Key ID	| Enter the secret key ID of the S3 bucket. |
+| Prefix / Folder | Enter the prefix / folder to store the traffic log. |
+
+4. Click **Save**.
+
+---
+
+### Fortiweb VM
+
+* [How-To export Attack, Traffic and Event logs to a log server](https://docs.fortinet.com/document/fortiweb/7.6.0/administration-guide/303842/logging).
```
Comment (ShirleyDenkberg) [inline Packs/FortinetFortiwebVM/README.md:88]:
```suggestion
* [How to export Attack, Traffic and Event logs to a log server](https://docs.fortinet.com/document/fortiweb/7.6.0/administration-guide/303842/logging).
```

File: Packs/FortinetFortiwebVM/README.md [hunk 7]
```diff
@@ -11,3 +22,94 @@ Fortiweb VM integration enables you to manage WAF policies and block cookies, UR
...
+
+**Enable Logging**
+1.  First, configure a SIEM Policy, before you can log the resource, you enable logging for the log type that you want to use as a trigger.
```
Comment (ShirleyDenkberg) [inline Packs/FortinetFortiwebVM/README.md:91]:
```suggestion
1.  First, configure a SIEM Policy. Before you can log the resource, you enable logging for the log type that you want to use as a trigger.
```

File: Packs/FortinetFortiwebVM/README.md [hunk 8]
```diff
@@ -11,3 +22,94 @@ Fortiweb VM integration enables you to manage WAF policies and block cookies, UR
...
+2.  **Log&Report** &rarr; **Log Config** &rarr; **Other Log Settings**.
+3.  Make sure that the Attack, Traffic and Event logs checkboxes are marked.
+4.  Click **Apply**.
+
+**Configure a SIEM Policy**
+1.  Before you can log to the resource, you enable logging for the log type that you want to use as a trigger.
+2.  Go to **Log&Report** **>** **Log Policy** &rarr; **SIEM Policy**.
```
Comment (ShirleyDenkberg) [inline Packs/FortinetFortiwebVM/README.md:98]:
```suggestion
2.  Go to **Log&Report** &rarr; **Log Policy** &rarr; **SIEM Policy**.
```

File: Packs/FortinetFortiwebVM/README.md [hunk 9]
```diff
@@ -11,3 +22,94 @@ Fortiweb VM integration enables you to manage WAF policies and block cookies, UR
...
+3.  For **Policy Name**, enter a unique name that other parts of the configuration can reference.
+4.  Click **Create New**, set the **Policy Type** to **ArcSight CEF**.
+5.  Input an IP address and port for the server.
+6.  Click **OK**.
+
+**Configure Log Settings**
+1. Go ti **Log&Report** &rarr; **Log Config** &rarr; **Global Log Settings**.
```
Comment (ShirleyDenkberg) [inline Packs/FortinetFortiwebVM/README.md:105]:
```suggestion
1. Go to **Log&Report** &rarr; **Log Config** &rarr; **Global Log Settings**.
```

**Comment by ShirleyDenkberg** [comment]
@eepstain Doc review completed.

**Review by OBavly — APPROVED** [review]

**Pull request merged**